en/docs/administer/logging-and-monitoring/logging/setting-up-logging-per-api.md

```diff
@@ -0,0 +1,62 @@
+---
+template: templates/swagger.html
+---
+# Setting up logging per API in API Manager
+
+!!! note
+    Enabling logs per API is only available after the WUM update with the timestamp "1600491803101"
```

Comment:
we usually give out the date the WUM was released. Can we add that info?

Reply:
Fixed